app/views/wikis/index.html.erb

```diff
@@ -8,20 +8,25 @@
 <% end %>
 
 <h4>My Wikis</h4>
-<table style="width:70%">
+<table style="width:50%">
```

Comment:
Try to avoid using inline `style='...'` attributes directly in HTML elements. Favor keeping all styles in the `app/assets/stylesheets/*.scss` stylesheet files for consistency.